src/exampleTypes/ResultTable.js

```diff
@@ -41,6 +41,84 @@ let popoverStyle = {
   userSelect: 'none',
 }
 
+let HighlightedColumn = withStateLens({ viewModal: false })(
+  observer(
+    ({
+      visibleFields,
+      node,
+      highlight = _.result('slice.0.highlight', getResults(node)),
+      header,
+      Cell = header ? 'th' : 'td',
+      Table = 'table',
+      Modal = null,
+      viewModal,
+    }) => {
+      let alsoHighlighted = _.difference(_.keys(highlight), visibleFields)
+      let getConnector = i => {
+        let length = alsoHighlighted.length
+        return i === length - 1 ? '' : i === length - 2 ? ' and ' : ', '
+      }
+      return highlight && !_.isEmpty(alsoHighlighted) ? (
+        <Cell key="highlight">
+          {Modal && (
+            <Modal isOpen={viewModal}>
+              <h3>Other Matching Fields</h3>
+              <Table>
+                <thead>
+                  <tr>
+                    {_.map(
+                      x => (
+                        <th key={x}>{_.startCase(x)}</th>
+                      ),
+                      alsoHighlighted
+                    )}
+                  </tr>
+                </thead>
+                <thead>
+                  <tr>
+                    {_.map(
+                      ([text]) => (
+                        <td>
+                          <TextHighlight
```

Comment:
You shouldn't need to do this. The fields come from elasticsearch already highlighted, just render the HTML from the server directly

Reply:
I can't render the HTML from the server directly since some of it has a whole HTML body and other structures :/ this is unsafe. We should clean it up first, then highlight.

Reply:
After talking with Sam about this:
> I'll just pass it along and see if it breaks, if it breaks I'll do the "remove all the other tags", and I'll do it in contexture elasticsearch

Reply:
Passing it is fine :)